.travis.yml

```diff
@@ -42,26 +42,19 @@ virtualenv:
   system_site_packages: true
 # command to install dependencies
 before_install:
-  - wget -O pyop2-ext.txt https://raw.github.com/OP2/PyOP2/master/requirements-ext.txt
-  - wget -O pyop2-git.txt https://raw.github.com/OP2/PyOP2/master/requirements-git.txt
+  - pip install -U pip
   - pip install -U pytest
-  - pip install psutil
-  - pip install cachetools
+  - pip install -U Cython
   - pip install pytest-benchmark
-  - "xargs -l1 pip install --allow-external mpi4py --allow-unverified mpi4py \
-       --allow-external petsc --allow-unverified petsc \
-       --allow-external petsc4py  --allow-unverified petsc4py \
-       < pyop2-ext.txt"
-  - "xargs -l1 pip install --allow-external mpi4py --allow-unverified mpi4py \
-       --allow-external petsc --allow-unverified petsc \
-       --allow-external petsc4py  --allow-unverified petsc4py \
-       < pyop2-git.txt"
-  - pip install -r requirements.txt
-  - if [[ $TRAVIS_PYTHON_VERSION == "2.6" ]]; then pip install argparse ordereddict; fi
+  - export BRANCH=$(git symbolic-ref --short -q HEAD)
+  - mkdir tmp
+  - cd tmp
 install:
-  - make
+  - ../scripts/firedrake-install --disable_ssh --global
+  - cd firedrake/src/firedrake
+  - git checkout $BRANCH
+  - python setup.py build_ext --inplace
```

Comment:
Should the install script optionally be able to install a branch?

If you run pytest from the parent directory, rather than the firedrake source directory, then it shouldn't find the checked out version, no.  Or do we do some horrible path insertion?